02_production.ipynb

```diff
@@ -29,7 +29,11 @@
    "cell_type": "markdown",
```

Comment:
<p>I don't think the "when appropriate" added here is a useful addition</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastbook/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>